src/Dotnet.Script.DependencyModel.Nuget/Dotnet.Script.DependencyModel.NuGet.csproj

```diff
@@ -8,7 +8,7 @@
     <RepositoryUrl>https://github.com/filipw/dotnet-script.git</RepositoryUrl>
     <RepositoryType>git</RepositoryType>
     <PackageTags>script;csx;csharp;roslyn;nuget</PackageTags>
-    <Version>0.2.0</Version>
+    <Version>1.0.0</Version>
```

Comment:
the version can be `0.3.0`. Before `1.0` it's not a violation of semver to have breaking changes (that's why scriptcs stayed on pre 1.0 so long 😀 )

Reply:
Actually, I did not know you could do that pre 1.0.0. :) Changing to 0.3.0 for both `DependencyModel` packages.